README.md

```diff
@@ -48,7 +48,7 @@ The following environment variables are used to launch the Red Hat AppStudio ins
 |---|---|---|---|
 | `GITHUB_TOKEN` | yes | A github token used to create AppStudio applications in github  | ''  |
 | `QUAY_TOKEN` | yes | A quay token to push components images to quay.io. Note the quay token must be your dockerconfigjson encoded in base64 format. Example: `export QUAY_TOKEN=$(base64 < ~/.docker/config.json)` | '' |
-| `DEFAULT_QUAY_ORG` | yes | A quay organization where repositories for component images will be created  | 'redhat-appstudio-qe'  |
+| `DEFAULT_QUAY_ORG` | yes | A quay organization where repositories for component images will be created  | ''  |
```

Comment:
I think the env `DEFAULT_QUAY_ORG` is not being used. There is a `DEFAULT_E2E_QUAY_ORG` but is a const

Reply:
`DEFAULT_QUAY_ORG` is used [here](https://github.com/redhat-appstudio/e2e-tests/blob/2f740a3c979060fb2ed280d1e9d79cf869553a57/magefiles/installation/install.go#L98)

`DEFAULT_E2E_QUAY_ORG` should be actually removed. I'll fix it in a following commit

Reply:
Sorry. I did have main branch updated on my side.